locker

```diff
@@ -7,39 +7,64 @@ function lock_image() {
 	echo $NAME@$DIGEST
 }
 
-function lock_dockerfile() {
+function unlock_image() {
+	IMAGE=$1
+	NAME=$(echo $IMAGE | grep '^[^@]\+' -o)
+	echo $NAME
+}
+
+function update_dockerfile() {
+	OPERATION=$1
+	shift
 	FILE=$1
 	LOCKFILE=$FILE.lock
 
 	while IFS= read -r LINE; do
 		if [[ $LINE =~ ^FROM ]]
 		then
 			IMAGE=$(echo "$LINE" | sed -r 's/^FROM +([^ ]+).*$/\1/')
-			LOCKIMAGE=$(lock_image $IMAGE)
+			LOCKIMAGE=$($OPTION $IMAGE)
```

Comment:
Should be `$OPERATION`.

Reply:
D'oh! Though tests seem to work with either...?